ee/main.go

```diff
@@ -118,8 +118,10 @@ func main() {
 	}
 
 	graphqlClient := graphql.NewGraphqlClient(
-		viper.GetString("jobSubmission.graphqlEndpoint"),
-		viper.GetString("jobSubmission.graphqlSecret"))
+		viper.GetString("graphqlEndpoint"),
+		viper.GetString("graphqlSecret"))
```

Comment:
Since we pulled these two configs, we should also update L316-L317 in `loadConfig()`, right?
```
        configs := []string{
		"customImage.pushSecretName",
		"jobSubmission.graphqlEndpoint",
		"jobSubmission.graphqlSecret",
```

Reply:
Added